src/components/Nav.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Route, Routes, NavLink } from 'react-router-dom';
+import RocketsList from './Rockets/RocketsList';
+import favicon from './Assets/saturn.png';
+import Profile from './Profile';
+import Missions from './missions/missions';
+
+const Nav = () => (
+  <>
+    <header className="header">
+      <div className="divIcon">
+        <img src={favicon} alt="fjstudio" height={50} width={50} />
+        <h1>Space Travel&apos;s Hub</h1>
+      </div>
+      <nav className="navbar">
+        <div className="nav-pills">
+          <NavLink className="link" to="/RocketsList">
+            Rockets
+          </NavLink>
+          <NavLink className="link" to="/Missions">
+            Missions
+          </NavLink>
+          <NavLink className="link" to="/Profile">
+            My profile
+          </NavLink>
```

Comment:
- [ ] Great job so far, in the design the button background color and link text color are the same please change the link color.
### Your website

![Screen Shot 2022-09-02 at 4 04 07 PM](https://user-images.githubusercontent.com/59416136/188151995-4e2c3acb-881e-4642-807b-c75003e7697a.png)

### Design

![Screen Shot 2022-09-02 at 4 04 43 PM](https://user-images.githubusercontent.com/59416136/188152085-13a25b8c-0f76-497b-bd8a-2af809161b24.png)

Reply:
This is well resolved! ✔️